pls_core/src/inflections/mod.rs

```diff
@@ -334,7 +334,7 @@ pub fn get_abbreviations_for_locale(
     let sql: String;
     if locale == "xx" {
         sql = "select name, description, '^' || name || '$' from _abbreviations".to_string();
```

Comment:
This doesn't sound right. Aren't we getting the latn string and adding ^$ to it. Let's discuss this please. I don't understand the syntax

Reply:
yeah, since the latn string is in the name column, I've handled the concatenation within the sql string.

Reply:
oh wow very cool!

Reply:
push through please when you are ready